sim/core/pet.go

```diff
@@ -220,6 +220,10 @@ func (pet *Pet) EnableWithTimeout(sim *Simulation, petAgent PetAgent, petDuratio
 	sim.AddPendingAction(pet.timeoutAction)
 }
 
+func (pet *Pet) SetStartattackDelay(startAttackDelay time.Duration) {
+	pet.startAttackDelay = startAttackDelay
+}
```

Comment:
Minor nit, call it `SetStartAttackDelay()` for consistency with the variable name.